consensus/enclave/impl/src/lib.rs

```diff
@@ -989,6 +989,13 @@ fn mint_output<T: Digestible>(
     amount: Amount,
     counter: usize,
 ) -> Result<TxOut> {
+    // Check if the token id makes sense right now
+    if !block_version.masked_token_id_feature_is_supported() && amount.token_id != 0 {
+        return Err(Error::FormBlock(
+            "Cannot mint outputs for non-MOB tokens until they are supported".to_string(),
```

Comment:
Optional/Equivalent: Note that they are not supported at this block version
```suggestion
            "Cannot mint outputs for non-MOB tokens at this block version".to_string(),
```